packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -75,15 +75,12 @@
 			<h4>Notebook</h4>
 		</section>
 		<template #preview>
-			<tera-drilldown-preview title="Preview">
-				<!-- TODO: pass dropdown info as prop to tera-drilldown-preview -->
-				<Dropdown
-					v-if="runList.length > 0"
-					:options="runList"
-					v-model="selectedRun"
-					option-label="label"
-					placeholder="Select a calibration run"
-				/>
+			<tera-drilldown-preview
+				title="Preview"
+				:options="outputs"
+				v-model:output="selectedOutputId"
+				@update:output="onUpdateOutput"
+			>
```

Comment:
BTW, not sure if you want to handle the checkbox functionality in here as well to handle the output nodes that show?  You can add the `is-selectetable` flag and handle the selection with the `@update:selection="onUpdateSelection"` event.  (see `tera-model-from-code`)
```
<tera-drilldown-preview
				:options="outputs"
				v-model:output="selectedOutputId"
				@update:output="onUpdateOutput"
				@update:selection="onUpdateSelection"
				:is-loading="isProcessing"
				is-selectable
			>
```

Reply:
Just added!